wp_api/src/parsed_url.rs

```diff
@@ -11,6 +13,20 @@ impl ParsedUrl {
     }
 }
 
+impl Display for ParsedUrl {
```

Comment:
I think this goes a little bit against the common practice of wrapper types implementing the `Display` of the inner type. Although that may be fine in some cases where it's obvious, I feel like in this one it can trip one up as they were expecting the [forwarded implementation of `Url`.](https://docs.rs/url/latest/url/struct.Url.html#impl-Display-for-Url)

Would using a different method for `ParsedUrl` address your needs? That way we can also make it available to `uniffi` clients.

Reply:
Easy enough to just have a separate method – see 15722a70934793825cc1a863c72f359bca07d7d4

Reply:
Oh yikes – I meant to do what was in your example and somehow had the wrong thing 🤦 

I've updated it to resolve this